app/models/concerns/pay/chargeable.rb

```diff
@@ -36,5 +36,40 @@ def stripe_refund!(amount)
       update(amount_refunded: amount)
     end
 
+    if defined?(Receipts::Receipt) && required_receipt_attributes?
+      def receipt
+        Receipts::Receipt.new(
+          id: id,
+          product: Pay.config.application_name,
+          company: {
+            name: Pay.config.business_name,
+            address: Pay.config.business_address,
+            email: Pay.config.support_email,
+          },
+          line_items: line_items,
+        )
+      end
+
+      def line_items
+        line_items = [
+          ["Date",           created_at.to_s],
+          ["Account Billed", "#{owner.name} (#{owner.email})"],
+          ["Product",        Pay.config.application_name],
+          ["Amount",         ActionController::Base.helpers.number_to_currency(amount / 100.0)],
+          ["Charged to",     "#{card_type} (**** **** **** #{card_last4})"],
```

Comment:
This will also need an update in the future, since we'll have PayPal as well. `card_type` will be `PayPal` and then `card_last4` will be the email address of the account.

This is only for Braintree functionality. I'm going to push up some code for their webhooks that I wrote yesterday.

Reply:
We can always extract this to a method that looks at the processor for the charge type and does it's thing - but that's for a later PR